### PR TITLE
SGX v4x updates

### PR DESCRIPTION
**Pull request by mythi**
Closes: #487

### CODE REVIEW

File: cmd/sgx_plugin/README.md [hunk 1]
```diff
@@ -63,11 +64,36 @@ The component has the same basic dependancies as the
...
+rules provided below to make the old device nodes as symlinks to the
+new device nodes.
+
+```bash
+$ cat /etc/udev/rules/9*.rules
+SUBSYSTEM=="misc",KERNEL=="enclave",MODE="0666"
+SUBSYSTEM=="misc",KERNEL=="sgx_enclave",MODE="0666",SYMLINK+="sgx/enclave"
+SUBSYSTEM=="sgx",KERNEL=="sgx/enclave",MODE="0666"
+SUBSYSTEM=="misc",KERNEL=="provision",GROUP="sgx_prv",MODE="0660"
+SUBSYSTEM=="misc",KERNEL=="sgx_provision",SYMLINK+="sgx/provision",GROUP="sgx_prv",MODE="0660"
```
Comment (ipuustin) [inline cmd/sgx_plugin/README.md:89]: Why different permissions for `sgx/enclave` and `sgx/provision`? Doesn't the `o` bit affect what the container workload can do, if it runs as a regular user? Also, I think symbolic links don't have a mode as such; they only can be changed by a user who has write access to the underlying directory.

Comment (mythi) [reply cmd/sgx_plugin/README.md]: @ipuustin no particular reason. I was using the rules mainly targeted for bare metal usage. I dropped the `GROUP` settings.

**Comment by mythi** [comment]
@ipuustin @rojkov ready for "official" review.

**Review by ipuustin — APPROVED** [review]
LGTM!

**Review by rojkov — APPROVED** [review]

**Pull request merged**